Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue

```diff
@@ -660,6 +674,30 @@ export default class CovidCardView extends Vue {
                             </v-btn>
                         </v-col>
                     </v-row>
+                    <v-row dense>
+                        <v-col class="text-right">
+                            <v-btn type="submit" class="mx-2 success">
+                                <span>START COVID THERAPY ASSESSMENT</span>
+                            </v-btn>
```

Comment:
Nit: the text for the button should use title case to match the others.  The uppercasing is applied by the button style.